### ISSUE TITLE
Accuracy Calculation Problem

### ISSUE THREAD

**Issue by whubaichuan** [IsmailKonak/FF-Algorithm-Pytorch-Implementation#1]
This accuracy calculation code seems to be wrong. 

```   
 def accuracy_f(self, y_pred, y_true):
        batch_size = y_pred.size(0)
        _, predicted_labels = y_pred.max(1)
        y_pred_onehot = torch.zeros_like(y_pred)
        y_pred_onehot.scatter_(1, predicted_labels.view(-1,1), 1)
        correct = (y_pred_onehot == y_true).sum().item()
        accuracy = correct / (batch_size * y_true.size(1))
        return accuracy
```

For a simple case, if batch_szie=1, we have 5 classes, y_pred= [0,0,0,0,1], y_true=[0,0,1,0,0], the accuracy here is 3/5. That is a big mistake.
